### PR TITLE
Implement set_layer method

### PR DESCRIPTION
**Pull request by JE-Random-Tech**
## Description
Implementation of the set_layer method.

* **Status**: *READY*

Branch              | PR
--------------------|-------
other_pr_production | [link]()
other_pr_master     | [link]()

## Todos
- [ ] Tests
- [ ] Documentation

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1.

### CODE REVIEW

**Comment by willprice** [comment]
Thanks @JE-Random-Tech ,
Can you add an entry to the `test_player_interface_commands` parameterised tests like https://github.com/willprice/python-omxplayer-wrapper/blob/master/tests/unit/test_omxplayer.py#L70

**Comment by JE-Random-Tech** [comment]
@willprice  Somehow the test for python 3.4 failed fetching http://deb.debian.org/debian/pool/main/p/python-pip/python-pip-whl_9.0.1-2_all.deb

**Comment by JE-Random-Tech** [comment]
> 
> 
> Use layer instead of alpha.

Thanks, sorry missed that.

**Comment by willprice** [comment]
> @willprice Somehow the test for python 3.4 failed fetching http://deb.debian.org/debian/pool/main/p/python-pip/python-pip-whl_9.0.1-2_all.deb

Yeah, seems like the travis config needs updating. Don't worry about it, I'll fix that on master once this is in. As long as python3.5+ pass we're good :+1:

File: omxplayer/player.py [hunk 1]
```diff
@@ -574,6 +574,17 @@ def set_position(self, position):
         self._player_interface.SetPosition(ObjectPath("/not/used"), Int64(position * 1000.0 * 1000))
         self.positionEvent(self, position)
 
+    @_check_player_is_active
+    @_from_dbus_type
+    def set_layer(self, layer):
+        """
+        Set the layer of the Video
```
Comment (willprice) [inline omxplayer/player.py]: Might be worth saying what the default layer is, and whether higher layers are above lower layers or vice versa

**Comment by JE-Random-Tech** [comment]
Thanks.

**Comment by willprice** [comment]
Thanks for your contribution :)

**Pull request merged**